core/server/routes/frontend.js

```diff
@@ -37,19 +38,22 @@ frontendRoutes = function () {
     });
 
     // Tags
-    router.get('/' + config.routeKeywords.tag + '/:slug/rss/', frontend.rss);
-    router.get('/' + config.routeKeywords.tag + '/:slug/rss/:page/', frontend.rss);
-    router.get('/' + config.routeKeywords.tag + '/:slug/' + config.routeKeywords.page + '/:page/', frontend.tag);
-    router.get('/' + config.routeKeywords.tag + '/:slug/', frontend.tag);
+    router.get('/' + routeKeywords.tag + '/:slug/rss/', frontend.rss);
+    router.get('/' + routeKeywords.tag + '/:slug/rss/:page/', frontend.rss);
+    router.get('/' + routeKeywords.tag + '/:slug/' + routeKeywords.page + '/:page/', frontend.tag);
+    router.get('/' + routeKeywords.tag + '/:slug/', frontend.tag);
 
     // Authors
-    router.get('/' + config.routeKeywords.author + '/:slug/rss/', frontend.rss);
-    router.get('/' + config.routeKeywords.author + '/:slug/rss/:page/', frontend.rss);
-    router.get('/' + config.routeKeywords.author + '/:slug/' + config.routeKeywords.page + '/:page/', frontend.author);
-    router.get('/' + config.routeKeywords.author + '/:slug/', frontend.author);
+    router.get('/' + routeKeywords.author + '/:slug/rss/', frontend.rss);
+    router.get('/' + routeKeywords.author + '/:slug/rss/:page/', frontend.rss);
+    router.get('/' + routeKeywords.author + '/:slug/' + routeKeywords.page + '/:page/', frontend.author);
+    router.get('/' + routeKeywords.author + '/:slug/', frontend.author);
+
+    // Post Live Preview
+    router.get('/' + routeKeywords.preview + '/:uuid', frontend.preview);
```

Comment:
While adding this, all the other lines of code looked too long to me :P I couldn't help myself